smalltalksrc/Melchor/VMBasicConstants.class.st

```diff
@@ -99,6 +99,7 @@ VMBasicConstants class >> namesDefinedAtCompileTime [
 	 will be emitted within #if defined(ANameDefinedAtCompileTime)...#endif."
 	^#(	VMBIGENDIAN
 		IMMUTABILITY
+		FEATURE_JIT_SIMD
```

Comment:
It would be nice to move this `VMBasicConstants class >> namesDefinedAtCompileTime` somewhere else, it looks like a strange dependency.